components/Dropdown/Dropdown.jsx

```diff
@@ -381,6 +381,7 @@ const Dropdown = ({
                       disabled={disabled}
                       text={_buttonLabel}
                       theme={theme}
+                      hasLabel
```

Comment:
to works in the Dropdown, we need to declare this const earlier.
`const hasLabel = !!label;` like you put on Input.jsx.

and here, we should put `hasLabel={hasLabel}` for the styled component receive the content of this constant.

After that, the DropInput has to receive the `hasLabel` and do the condition, the same way you did on Input too: `margin-top: ${hasLabel ? xsmall : '0'}px;`